extensions/wikia/Search/templates/WikiaSearch_CrossWiki_result.php

```diff
@@ -14,9 +11,9 @@
 		<p class="description"><?= $description; ?></p>
 
 		<ul class="wiki-statistics subtle">
-			<li><?= $pagesMsg ?></li>
-			<li><?= $imgMsg ?></li>
-			<li><?= $videoMsg ?></li>
+			<li><?= $pages['count'] ?> <?= $pages['message'] ?></li>
```

Comment:
This is in general a bad practice for i18n, the number should be included in the message so the translators have control over the order of the number and the message.